server/src/test/scala/controllers/UsersControllerSpec.scala

```diff
@@ -526,4 +528,59 @@ class UsersControllerSpec extends PlayPostgresSpec with LoginUtils {
         error must be("The given email/password doesn't match")
     }
   }
+
+  "POST /users/email-verification-token" should {
+    "success on send verifying token user's email" in withApiClient { client =>
+      val name = Name.trusted("wiringbits")
+      val email = Email.trusted("test1@email.com")
+      val request = SendEmailVerificationToken.Request(
+        email = email,
+        captcha = Captcha.trusted("test")
+      )
+
+      val userRequest = CreateUser.Request(
+        name = name,
+        email = email,
+        password = Password.trusted("test123..."),
+        captcha = Captcha.trusted("test")
+      )
+
+      val verificationToken = UUID.randomUUID()
+      when(tokenGenerator.next()).thenReturn(verificationToken)
+      client.createUser(userRequest).futureValue
+      
+      val response = client.sendEmailVerificationToken(request).futureValue
+      response.expiresAt mustNot be (null)
```

Comment:
This assertion is unnecessary because this Instant can't be null and even if it is, in Scala we try to avoid comparing a value to null. Instead of this you can check if the user token was created

Reply:
A simple way to verify this is:
- Let's get the current time before executing the request
- Assert that the `expiresAt` time is later than the previously computed time.

While we could check for the expected expiration time, you will need to do some mocking to the clock which I won't ask you for in this PR.